src/ui/views/TradingBlock.tsx

```diff
@@ -6,6 +6,10 @@ import { getCols, helpers, toWorker } from "../util";
 import { DataTable, PlayerNameLabels } from "../components";
 import type { View, ThenArg } from "../../common/types";
 import type api from "../../worker/api";
+import TradeFilter, { filterType } from "../views/TradeFilter/TradeFilter";
+import { AnimatePresence, motion } from "framer-motion";
```

Comment:
Whoa you even animated it??

Reply:
hah indeed, but I only took a cursory look at the library so feel free to change it to a more appropriate animation